### PR TITLE
Doc: Add liquid examples

### PR DESCRIPTION
**Pull request by eloycoto**
Signed-off-by: Eloy Coto <eloy.coto@acalustra.com>

### CODE REVIEW

File: doc/liquid_conditions.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# Liquid conditions examples
+
+When using liquid, a lot of things can be achieved, and here are some examples
```
Comment (porueesq) [inline doc/liquid_conditions.md:3]: You can use Liquid to configure conditions applicable across multiple policies to <insert_purpose>

Some questions to help you:
* What's the benefit of using Liquid to configure conditions across multiple policies?
* Are there any prerequisites to use these examples?
* Are there any limitations with these examples? For instance, can these examples be used in a production environment?

File: doc/liquid_conditions.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+of conditions that can be used across multiple policies
+
+
+## Headers
+
+### Header is not present
+
+Condition pass if header 'Backend' is sent and it's staging
```
Comment (porueesq) [inline doc/liquid_conditions.md:11]:
```suggestion
Condition passes if the  'Backend' header is sent and it's staging
```

Questions:
a. Who/What sends the  'Backend' header ?
b. What does `staging` refer to?

File: doc/liquid_conditions.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+         "left":" {% if headers['Backend'] %}{{ headers['Backend'] }}{%else%}Notpresent{%endif%}",
+         "left_type":"liquid",
+         "op":"==",
+         "right":"staging"
+      }
+   ]
+}
+```
+
+If want to test when the header is not present, the condition will be like this:
```
Comment (porueesq) [inline doc/liquid_conditions.md:26]:
```suggestion
If you want to test when the `Backend` header is not present, the condition is as follows:
```

File: doc/liquid_conditions.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+         "op":"==",
+         "right":"Notpresent"
+      }
+   ]
+}
+```
+
+## Numeric operations
+
+These are harder, by default, all the values on liquid are strings, so we need
```
Comment (porueesq) [inline doc/liquid_conditions.md:43]:
```suggestion
Numeric operations require additional steps. By default, all the values on liquid are strings, so you need
```

File: doc/liquid_conditions.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+to get any integer before do any operation. We can use `minus` or `plus`
```
Comment (porueesq) [inline doc/liquid_conditions.md:44]:
```suggestion
to get any integer before do any operation. Use `minus` or `plus`
```

File: doc/liquid_conditions.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+filters using 0, and do something like this:
```
Comment (porueesq) [inline doc/liquid_conditions.md:45]:
```suggestion
filters along with `0` (zero), and apply the following configuration:
```

File: doc/liquid_conditions.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+    }
+ ]
+}
+```
+
+@TODO: something like a int filter should be added.
+@TODO: default filter should be added too.
+
+
+## Another examples:
```
Comment (porueesq) [inline doc/liquid_conditions.md:65]:
```suggestion
## Additional examples:
```

**Review by porueesq — CHANGES_REQUESTED** [review]
Some questions and suggestions

**Review by eguzki — APPROVED** [review]

**Pull request merged**